src/main/scala/org/apache/mesos/chronos/notification/JobNotificationObserver.scala

```diff
@@ -0,0 +1,43 @@
+package org.apache.mesos.chronos.notification
+
+import java.util.logging.Logger
+
+import akka.actor.ActorRef
+import com.google.inject.Inject
+import org.apache.mesos.chronos.scheduler.jobs._
+import org.joda.time.{DateTimeZone, DateTime}
+
+class JobNotificationObserver @Inject()(val notificationClients: List[ActorRef] = List(),
+                                      val clusterName: Option[String] = None) extends JobsObserver {
+  private[this] val log = Logger.getLogger(getClass.getName)
+  val clusterPrefix = clusterName.map(name => s"[$name]").getOrElse("")
+
+  override def onEvent(event: JobEvent): Unit = event match {
+    case JobRemoved(job) => sendNotification(job, "%s [Chronos] Your job '%s' was deleted!".format(clusterPrefix, job.name), None)
+    case JobDisabled(job, cause) => sendNotification(
+      job,
+      "%s [Chronos] job '%s' disabled".format(clusterPrefix, job.name),
+      Some(cause))
+
+    case JobRetriesExhausted(job, taskStatus, attempts) =>
+      val msg = "\n'%s'. Retries attempted: %d.\nTask id: %s\n"
+        .format(DateTime.now(DateTimeZone.UTC), job.retries, taskStatus.getTaskId.getValue)
+      sendNotification(job, "%s [Chronos] job '%s' failed!".format(clusterPrefix, job.name),
+        Some(TaskUtils.appendSchedulerMessage(msg, taskStatus)))
+
```

Comment:
Would a generic log.info() for any observer that does not handle a specific case suffice?